R/stat-smooth.r

```diff
@@ -38,7 +40,7 @@ stat_smooth <- function(mapping = NULL, data = NULL,
                         geom = "smooth", position = "identity",
                         ...,
                         method = "auto",
-                        formula = y ~ x,
+                        formula = NULL,
```

Comment:
I also now realise that this makes `method` and `formula` inconsistent. We should probably also default `method` to `NULL` (following https://principles.tidyverse.org/def-inform.html) and also allow `"auto"` for backward compatibility.